prog/vnet/subnet_nexus.rb

```diff
@@ -21,8 +21,9 @@ def self.assemble(project_id, name: nil, location: "hetzner-hel1", ipv6_range: n
       ps = PrivateSubnet.create(name: name, location: location, net6: ipv6_range, net4: ipv4_range, state: "waiting") { _1.id = ubid.to_uuid }
       ps.associate_with_project(project)
       port_range = allow_only_ssh ? 22..22 : 0..65535
-      fw = Firewall.create_with_id(private_subnet_id: ubid.to_uuid, name: "#{name}-default")
+      fw = Firewall.create_with_id(name: "#{name}-default")
       ["0.0.0.0/0", "::/0"].each { |cidr| FirewallRule.create_with_id(firewall_id: fw.id, cidr: cidr, port_range: Sequel.pg_range(port_range)) }
+      fw.associate_with_private_subnet(ps, apply_firewalls: false)
```

Comment:
asking to understand; why we don't want to apply firewall here?

Reply:
When we provision the VM, initially, we'll already apply the firewalls fresh, we don't need to increment the semaphore, here.